pkg/api/converterfromosaapi.go

```diff
@@ -4,10 +4,10 @@ import (
 	"github.com/openshift/openshift-azure/pkg/api/v1"
 )
 
-// ConvertVLabsOpenShiftClusterToContainerService converts from a
-// v1.OpenShiftCluster to a ContainerService.
-func ConvertVLabsOpenShiftClusterToContainerService(oc *v1.OpenShiftCluster) *ContainerService {
-	cs := &ContainerService{
+// ConvertV1OpenShiftManagedClusterToOpenShiftCluster converts from a
+// v1.OpenShiftManagedCluster to a OpenShiftCluster.
+func ConvertV1OpenShiftManagedClusterToOpenShiftCluster(oc *v1.OpenShiftManagedCluster) *OpenShiftCluster {
```

Comment:
Still different?

Reply:
Not following now.  Are you asking to remove the types from the name completely?  I was thinking you were asking to update the types in the name itself.  We can't use the same function name, the files are in the same package.

Reply:
Why is `v1.OpenShiftManagedCluster` different from `OpenShiftCluster`? Shouldn't the former be `v1.OpenShiftCluster` or the latter `OpenShiftManagedCluster`?

Reply:
ohhhh, I see what you're asking now.  External is named that way because of

> Public name of the resource is going to be openShiftManagedClusters. This: https://github.com/openshift/openshift-azure/blob/master/pkg/api/v1/types.go#L5 should be updated to: OpenShiftManagedCluster

https://github.com/openshift/openshift-azure/issues/149

We could use the same internal if we wanted to, I have no strong opinion there and was just keeping with the convention that they were different.

Reply:
I would prefer to use the same name internally. Can be a follow-up.